### PR TITLE
MM-1899 Enabled and added help text for settings items

### PR DESCRIPTION
**Pull request by nickago**

### CODE REVIEW

File: web/react/stores/channel_store.jsx [hunk 1]
```diff
@@ -270,4 +270,5 @@ ChannelStore.dispatchToken = AppDispatcher.register(function(payload) {
     }
 });
 
+ChannelStore.setMaxListeners(11);
```
Comment (crspeller) [inline web/react/stores/channel_store.jsx]: What's this about? Seems unrelated to ticket.

**Comment by crspeller** [comment]
@asaadmahmoodspin This probably needs better formatting like MM-2035

File: web/react/stores/channel_store.jsx (hunk 1 above)
Comment (nickago) [inline web/react/stores/channel_store.jsx]: I needed to up the max allowed listeners so I could put a channel store listener on the notifications tab

Comment (crspeller) [inline web/react/stores/channel_store.jsx]: So the default max is 10? Any idea why that is?

Comment (nickago) [inline web/react/stores/channel_store.jsx]: https://nodejs.org/api/events.html#events_emitter_setmaxlisteners_n

Apparently it's the perfect number for dealing with memory leaks

Comment (crspeller) [inline web/react/stores/channel_store.jsx]: Alright cool.

**Comment by crspeller** [comment]
+1

**Comment by rgarmsen2295** [comment]
+1 from me, looks good

**Comment by nickago** [comment]
UI changes added courtesy of @asaadmahmoodspin and signed off on
